compose/cli/main.py

```diff
@@ -290,7 +291,8 @@ def build(self, options):
             build_args=build_args,
             gzip=options.get('--compress', False),
             parallel_build=options.get('--parallel', False),
-            silent=options.get('--quiet', False)
+            silent=options.get('--quiet', False),
+            _exec=bool(options.get('--exec', False)),
```

Comment:
here as well (I think)

Reply:
Perhaps other locations as well, but I'm not a python coder 😅 and not super-familiar with the codebase (just noticed the PR)

Reply:
sorted!